packages/features/embed/lib/EmbedCodes.tsx

```diff
@@ -118,29 +127,39 @@ export const Codes = {
       uiInstructionCode: string;
       previewState: PreviewState;
       namespace: string;
+      theme: string;
     }) => {
       return code`${getApiNameForVanillaJsSnippet({ namespace, mainApiName: "Cal" })}("inline", {
-	elementOrSelector:"#my-cal-inline",
-	calLink: "${calLink}",
-	layout: "${previewState.layout}"
```

Comment:
`layout` was supposed to be inside config. This change automatically fixes the bug.